core/src/main/java/com/rudderstack/android/sdk/core/RudderMessage.java

```diff
@@ -35,7 +35,7 @@
     @SerializedName("userProperties")
     private Map<String, Object> userProperties;
     @SerializedName("integrations")
-    private Map<String, Boolean> integrations = new HashMap<>();
+    private Map<String, Object> integrations = new HashMap<>();
```

Comment:
can we initialize `integrations` with null so that if RudderOptions is not used, we don't create an empty list every time.

Reply:
done